libmamba/include/mamba/core/context.hpp

```diff
@@ -197,6 +197,7 @@ namespace mamba
         bool no_env = false;
 
         std::size_t lock_timeout = 0;
+        bool disable_lockfile = false;
```

Comment:
IMO we should negative this boolean (and option) like here https://github.com/mamba-org/mamba/blob/e0527eb061b0ce724dcb3c2f34db25c907ee7113/libmamba/include/mamba/core/transaction_context.hpp#L51 https://github.com/mamba-org/mamba/blob/e0527eb061b0ce724dcb3c2f34db25c907ee7113/micromamba/src/common_options.cpp#L341-L343

Reply:
@wolfv WDYT?

Reply:
If that is what you want I will change it.  It's your choice.